src/main/java/net/dv8tion/jda/api/events/message/MessageReceivedEvent.java

```diff
@@ -39,17 +40,22 @@
  *     <li>{@link net.dv8tion.jda.api.requests.GatewayIntent#GUILD_MESSAGES GUILD_MESSAGES} to work in guild text channels</li>
  *     <li>{@link net.dv8tion.jda.api.requests.GatewayIntent#DIRECT_MESSAGES DIRECT_MESSAGES} to work in private channels</li>
  * </ul>
+ *
+ * @deprecated This event has been renamed to match standard Discord event naming.
+ *             Interaction with this event is "redirected" to {@link net.dv8tion.jda.api.events.message.MessageCreateEvent MessageCreateEvent}.
+ *             In fact, it's now a container for MessageCreateEvent.
 
  */
-//TODO-v5: Consider renaming this to MessageCreateEvent to match standard event naming (and to match Discord naming!)
+@Deprecated
+@ReplaceWith("MessageCreateEvent")
 public class MessageReceivedEvent extends GenericMessageEvent
 {
-    private final Message message;
+    private final MessageCreateEvent event;
```

Comment:
Why'd you change this? It doesn't make a whole lot of sense to me

Reply:
> ...so that when making changes to logic, we don't need to edit same code in 2 places.

* When adding a new method to `MessageCreateEvent`, we don't need to copy-paste the code, we just create a method of the same name in the `MessageReceivedEvent` and call the method from the new event in it.
* When editing the code, the same thing.

It seemed right to me. I think it's correct (even if it's only getters so far) to take out their logic of work to the `MessageCreateEvent`. 

And here I really don't know: if after "deprecate" the old event will not be supported in ANY way (no adding/removing logic/fixes (for backward compatibility)), then this thing is really useless.